### PR TITLE
feat(changelog): iam-added-secret-manager-is-available 2023-03-06

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Lucas Nilsson

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Secret manager is available in Public Beta 
+status: added
+author:
+    fullname: 'secret-manager'
+    url: 'https://slack.scaleway.com'
+date: 3/6/2023
+category: identity-and-access-management
+product: iam
```
Comment (ofranc) [inline changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx:9]: secret-manager non ?

File: changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: Secret manager is available in Public Beta 
+status: added
+author:
+    fullname: 'secret-manager'
+    url: 'https://slack.scaleway.com'
+date: 3/6/2023
```
Comment (nedaehm) [inline changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx:7]:
```suggestion
date: 2023-03-06
```

**Review by bene2k1 — APPROVED** [review]

File: changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: Secret Manager is available in Public Beta 
+status: added
+author:
+    fullname: 'secret-manager'
+    url: 'https://slack.scaleway.com'
+date: 2023-03-06
+category: identity-and-access-management
```
Comment (nedaehm) [inline changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx:8]:
```suggestion
category: security-identity
```

File: changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+product: iam
```
Comment (nedaehm) [inline changelog/march2023/2023-03-06-iam-added-secret-manager-is-available.mdx:9]:
```suggestion
product: secret-manager
```

**Pull request merged**